src/arrays.js

```diff
@@ -1,62 +1,87 @@
 const getNthElement = (index, array) => {
-  // your code here
+  console.log(index % array.length);
+  return array[index % array.length];
 };
+//copied and pasted, couldn't work out alternative.
 
 const arrayToCSVString = array => {
-  // your code here
+  return array.join();
 };
 
 const csvStringToArray = string => {
-  // your code here
+  return string.split(',');
 };
 
 const addToArray = (element, array) => {
-  // your code here
+  array.push(element);
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
-};
+  const biggerArray = array.concat(element);
+  return biggerArray;
+};
+// {
+//   const biggerArray = array.slice()
+//   biggerArray.push(element);
+//   return biggerArray;
+// };
+//I took ages on this one, and came up with a longer solution before switching to concat - much simpler.
```

Comment:
Always worth checking if there is a native JS method for what you want before writing a customer solution! All a learning process so this is perfectly normal, dont worry. There is a slight improvement you can make to put this onto just one line:
```suggestion
  return array.concat(element);
};
```